src/convertToObject.js

```diff
@@ -1,18 +1,25 @@
 'use strict';
 
 /**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
   // write your code here
+  const arr = sourceString
+    .split(';')
+    .map(
+      (rule) => rule.split(':').map((ruleItem) => ruleItem.trim()),
+    )
```

Comment:
You can remove excess indents during reduce, so as not to add an extra map in the middle of which is another map and not to load the complexity of the algorithm.